setup.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 by Delphix. All rights reserved.
+#
+
+from setuptools import find_packages
+from setuptools import setup
+from src.main import VERSION
+
+# Get the long description from the README file
+long_description = "This is an executable for Delphix New Relic Integration"
```

Comment:
For standardization stake, call it "Delphix Integration for New Relic"

We are trying to have similar structure for the others..
Delphix Spoke for ServiceNow
Delphix Provider for Terraform
Delphix plugin for Jenkins
etc

Reply:
sure